packages/tables/frontend/sirius-components-tables/src/table/TableContent.tsx

```diff
@@ -44,14 +29,20 @@ export const TableContent = memo(({ editingContextId, representationId, table, r
     editDisplayMode: 'cell',
     enableEditing: !readOnly,
     enableStickyHeader: true,
+    muiTableBodyProps: {
```

Comment:
Where is the stripeRow option used on the frontend? Did I miss something because this seems very much hardcoded and not optional at all. Did you just copy and paste the sample from Material React Table and called it a day?

Reply:
Indeed, I missed something during a rebase, sorry.